osbs/conf.py

```diff
@@ -270,6 +280,10 @@ def get_builder_use_auth(self):
                                default=self.get_use_auth(),
                                is_bool_val=True)
 
+    def get_builder_openshift_url(self):
+        """ url of OpenShift where builder will connect """
+        return self._get_value("builder_openshift_url", self.conf_section, "builder_openshift_url")
```

Comment:
Should we perhaps fall back to the value of `openshift_url` if `builder_openshift_url` is not defined?